README.md

```diff
@@ -1,61 +1,87 @@
 # U6143_ssd1306
-## Preparation
+
+##  I2C
+Begin by enabling the I2C interface:
+
 ```bash
 sudo raspi-config
 ```
-Choose Interface Options 
-Enable i2c
+
+Choose `Interface Options` | `I2C`, then answer `Yes` to whether you would like the ARM I2C interface to be enabled.
+
+Install Git and library dependencies
+
+```bash
+sudo apt update
+sudo apt install git wiringpi
+```
 
 ##  Clone U6143_ssd1306 library 
 ```bash
 git clone https://github.com/UCTRONICS/U6143_ssd1306.git
 ```
+
 ## Compile 
 ```bash
 cd U6143_ssd1306/C
 ```
 ```bash
-sudo make clean && sudo make 
+make clean && make 
 ```
+
 ## Run 
 ```
 sudo ./display
 ```
 
 ## Add automatic start script
-- Open the rc.local file 
+Copy the binary file to `/usr/local/bin/`:
+
 ```bash
-sudo nano /etc/rc.local
+sudo cp ./display /usr/local/bin/
 ```
-- Add command to the rc.local file
+
+Choose one of the following configuration options (`systemd` **or** `rc.local`):
+ 
 ```bash
-cd /home/pi/U6143_ssd1306/C
-sudo make clean 
-sudo make 
-sudo ./display &
+sudo cp ./contrib/U6143_ssd1306.service /etc/systemd/system/
+sudo systemctl daemon-reload
+sudo systemctl enable U6143_ssd1306.service
+sudo systemctl start U6143_ssd1306.service
```

Comment:
This could be made one line shorter with `sudo systemctl enable --now U6143_ssd1306.service`.

Reply:
Thanks for the suggestion, and indeed there's a more efficient way to do that, but going to leave that as a "won't fix" - there's more than one way to skin a cat, and the intent is to show the most common/familiar way to do things, not necessarily the shortest.